### PR TITLE
Add mulw (multiple wide) TEAL command

### PR DESCRIPTION
**Pull request by pzbitskiy**
* return low (top of the stack) and high uint64 values of the result

### CODE REVIEW

File: data/transactions/logic/eval.go [hunk 1]
```diff
@@ -585,6 +588,36 @@ func opMul(cx *evalContext) {
 	cx.stack = cx.stack[:last]
 }
 
+func opMulwImpl(x, y uint64) (uint64, uint64, error) {
```
Comment (algobolson) [inline data/transactions/logic/eval.go]: it would be nice to name the return fields (high, low, err)

File: data/transactions/logic/eval.go [hunk 2]
```diff
@@ -585,6 +588,36 @@ func opMul(cx *evalContext) {
...
+
+	return high.Uint64(), low.Uint64(), nil
+}
+
+func opMulw(cx *evalContext) {
+	last := len(cx.stack) - 1
+	prev := last - 1
+	high, low, err := opMulwImpl(cx.stack[prev].Uint, cx.stack[last].Uint)
+	if err != nil {
+		cx.err = errors.New("mulw overflowed")
```
Comment (algobolson) [inline data/transactions/logic/eval.go]: `cx.err = err`

**Review by algobolson — CHANGES_REQUESTED** [review]
nice, pretty clean, a couple small tweaks to go code style

File: data/transactions/logic/eval.go (hunk 2 above)
Comment (pzbitskiy) [reply data/transactions/logic/eval.go]: done

File: data/transactions/logic/eval.go (hunk 1 above)
Comment (pzbitskiy) [reply data/transactions/logic/eval.go]: done

**Pull request merged**